store/root.js

```diff
@@ -44,6 +44,13 @@ class Root extends Store {
     },
   }
 
+  constructor(options) {
+    super(null, {
+      ...options,
+      reqeust: (args) => Root.defaultOptions.request(...args),
```

Comment:
```suggestion
      request: (args) => Root.defaultOptions.request(...args),
```

Reply:
Because there is another typo:

```suggestion
      request: (...args) => Root.defaultOptions.request(...args),
```

Reply:
Good catch but sadly it's not enough.

Reply:
If it's not, I will bind the function. This will solve the problem